tests/functions/module_web.test.ts

```diff
@@ -0,0 +1,28 @@
+import { evaluateCellText } from "../test_helpers/helpers";
+
+describe("HYPERLINK formula", () => {
+  test("The evaluated result is the link", () => {
+    expect(evaluateCellText("A1", { A1: '=HYPERLINK("https://www.odoo.com", "Odoo")' })).toBe(
+      "Odoo"
+    );
+    expect(evaluateCellText("A1", { A1: '=HYPERLINK("https://www.odoo.com")' })).toBe(
+      "https://www.odoo.com"
+    );
+    expect(evaluateCellText("A1", { A1: "=HYPERLINK(A2)", A2: "https://www.odoo.com" })).toBe(
+      "https://www.odoo.com"
+    );
+    expect(
+      evaluateCellText("A1", { A1: '=HYPERLINK(A2, "Odoo")', A2: "https://www.odoo.com" })
+    ).toBe("Odoo");
+    expect(
+      evaluateCellText("A1", { A1: "=HYPERLINK(A2, A3)", A2: "https://www.odoo.com", A3: "Odoo" })
+    ).toBe("Odoo");
+    expect(evaluateCellText("A1", { A1: '=HYPERLINK("invalidUrl")' })).toBe("invalidUrl");
+    expect(evaluateCellText("A1", { A1: '=HYPERLINK("")' })).toBe("[]()");
+    expect(evaluateCellText("A1", { A1: '=HYPERLINK(" ")' })).toBe(" ");
+    expect(evaluateCellText("A1", { A1: "=HYPERLINK(2)" })).toBe("2");
+    expect(evaluateCellText("A1", { A1: "=HYPERLINK(true)" })).toBe("TRUE");
+    expect(evaluateCellText("A1", { A1: '=HYPERLINK("1/31/2022")' })).toBe("1/31/2022");
```

Comment:
add the same tests with the *label* being a number, boolean, date, .... (and a normal url)